### DISCUSSION TITLE
Q: do these files need to be checked in? The build process changes them in the source tree

### DISCUSSION THREAD

**Discussion by rvjansen** [adesutherland/CREXX#267] [Q&A]
```	
modified:   re2c/bootstrap/src/parse/parser.cc
modified:   re2c/bootstrap/src/parse/parser.h
```
These are modified in the source tree, and because they contain a location dependent comment, they make trouble with the merge process. Should they not only be written to in the build tree?

**Comment by adesutherland** [comment]
The re2c build comes from the re2c project  - so I am not sure we should change it here but would have to get re2c to fix it upstream ...?

**Reply by rvjansen** [reply]
But they are being rewritten with the branch name in comments - in the source tree, not the build tree. I need to figure out how we can change this in the cmakelists

**Reply by rvjansen** [reply]
Also, this is the root of the problems in the 'machine' subdirectory that sometimes occur. These are also modified in the source tree. Until now I did a new clone and checkout, but only some .h files need to be removed:
```
-rw-r--r--  1 rvjansen  staff   907B Feb 12 22:50 CMakeLists.txt
-rw-r--r--  1 rvjansen  staff    22K Mar  5 20:03 instrmiss.h
-rw-r--r--  1 rvjansen  staff    22K Mar  5 20:03 instrset.h
-rw-r--r--  1 rvjansen  staff   6.8K Feb 26 21:04 rxvmigen.rexx
-rw-r--r--  1 rvjansen  staff    27K Mar  5 20:01 rxvminst.c
-rw-r--r--  1 rvjansen  staff   1.6K Feb 12 22:50 rxvminst.h
```
so instrset.h and instrmiss.h need to be gone if you encounter this problem. rxvminst.c needs to stay.

**Reply by rvjansen** [reply]
opened https://github.com/adesutherland/CREXX/issues/273
